### PR TITLE
Bump PoE contracts to v0.11.0

### PR DESCRIPTION
**Pull request by alpe**
After #395 
See #377

### CODE REVIEW

File: x/poe/contract/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-v0.10.0
+v0.11.0
```
Comment (pinosu) [inline x/poe/contract/version.txt:1]: 👍

**Review by pinosu — APPROVED** [review]
lgtm 💯

**Comment by ethanfrey** [comment]
Yay! Everything passes simulations now!

**Review by ethanfrey — APPROVED** [review]
lgtm

**Pull request merged**